source/user-manual/capabilities/vulnerability-detection/how_it_works.rst

```diff
@@ -17,31 +17,42 @@ The global vulnerability database is created automatically, currently pulling da
-This improves the performance and avoids repeated alerts during a configurable time. We have then three different types of scan:
+The `Vulnerability Detector` module performs a scan when it is enabled generating a vulnerabilities inventory of packages and operating system installed in the managed agents.
+It also can run a scan on startup (:ref:`run_on_start <vuln_det_run_on_start>`) and every certain period of time (:ref:`interval <vuln_det_interval>`).
+In any of these cases, the packages and the operating system that were already scanned will not be re-scanned, only the new ones will do and alerts are going to be generated.
+If the database of vulnerabilities receives new CVEs information and :ref:`min_full_scan_interval <vuln_det_min_full_scan_interval>` expires all packages as well as the operating system will be re-scanned.
+Reducing the times that the `Full scans` are triggered improves the usage of the system resources and increase the performance avoiding repeat alerts of already notified vulnerabilities during a configurable time.
 
-- Baseline: This scan type will be triggered the first time after Vulnerability Detector is enabled. It performs a full scan for every single package installed as well as the operating system. As a result, the CVEs inventory will have the information of the vulnerabilities detected but no alerts will be generated.
-- Full scan: In this scan type, every single package installed as well as the operating system are scanned. It runs only when the configured :ref:`min_full_scan_interval <vuln_det_min_full_scan_interval>` expires and the CVEs database is updated with new information. As a result, the newly detected vulnerabilities and the ones that no longer affect the agent will be alerted.
-- Partial scans: Only new packages are scanned and the vulnerabilities detected are alerted.
+For more details we have then three different scan types that are described below.
+
+- `Baseline`: This scan type will be triggered the first time after `Vulnerability Detector` is enabled. It performs a full scan for every single package installed as well as the operating system. As a result, the CVEs inventory will have the information of the vulnerabilities detected but no alerts will be generated.
+- `Full scan`: In this scan type, every single package installed as well as the operating system are scanned. It runs only when the configured :ref:`min_full_scan_interval <vuln_det_min_full_scan_interval>` expires and the CVEs database is updated with new information. As a result, the newly detected vulnerabilities and the ones that no longer affect the agent will be alerted.
+- `Partial scan`: Only new packages are scanned and the new vulnerabilities detected are alerted together with deleted or updated packages.
```

Comment:
We should add here that removed vulnerabilities will be reported, as it is in full_scan.

Reply:
Or, we can refer in both scans to report **"vulnerabilities update"** wrapping the addition and the deletion of vulnerabilities.
It will be explained in more detail in the **Alerts generation** section.